client/src/components/ExercisesCategory.js

```diff
@@ -11,24 +12,16 @@ export default function ExercisesCategory({ exercises }) {
     <div>
       <h1>Exercise Category</h1>
       {filteredExercises.map((exercise) => (
-        <p>{exercise.titel}</p>
+        <Link to={'/Uebungen_Ueberblick/' + category + '/' + exercise._id}>
+          <p>{exercise.titel}</p>
+        </Link>
```

Comment:
When category is undefined, the resultng url will be `/Uebungen_Ueberblick/undefined/<id>`, not sure if this is what you want?

Don't forget to assign a `key`